src/test.js

```diff
@@ -771,6 +771,7 @@ export function internalStop( test ) {
 					internalRecover( test );
 				};
 			};
+			clearTimeout( config.timeout );
```

Comment:
Moved here so that there is always a direct pair of `clearTimeout( config.timeout )` and `config.timeout = …`.